### ISSUE TITLE
We need to improve the flow for setting up a project.

### ISSUE THREAD

**Issue by jferragut** [learnpack/learnpack#173]

**Comment by jferragut** [comment]
# Current Workflow
1. install globals `npm i -g learnpack` or in linux `sudo npm i -g learnpack`
2. initalize `learnpack init`
3. Select autograding: incremental, isolated, no grading
4. select a title
5. select description
6. set difficulty
7. select avg time to complete
8. start a server `learnpack start`
9. sample lesson is created "01-hello-world"

**Comment by jferragut** [comment]
@alesanchezr are we planning on non-technical people using? I recall us discussing that this was the goal. If so, it looks like node setup and install would have to be covered for the novice or non-techie. 

I see that being a roadblock. Other than that, this should just be documented really well. The new process you added with folder structure appears to be easy enough for the time being.

**Comment by alesanchezr** [comment]
Yes we can have a learnpack tutorial on how tonuse learnpack, with videos 🤯
